src/content/docs/tutorial-external-services/respond-external.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: Respond to API issues
+metaDescription: Learn how to manage APIs and other external services
+---
+
+import externalMaps from 'images/apm_screenshot-crop_external-maps.webp'
+
+import externalRPM from 'images/apm_screenshot-crop_maps-rpm.webp'
+
+import externalOverview from 'images/apm_screenshot-crop_maps-home.png'
+
+Almost all applications and services make calls to APIs and other external services. When something as simple as pressing the `buy` button on a site can make dozens of API calls, issues are bound to arise. Problematic API calls can result in anything from a simple inventory mishap to a code-red situations where your site can't communicate with your payment processor. 
```

Comment:
Repetition of the word simple. Also, opening with the word almost feels a little passive. Can you rephrase this to make it more direct? That's nit-picky so feel free to disregard.